roles/kubernetes/secrets/templates/openssl-node.conf.j2

```diff
@@ -0,0 +1,26 @@
+{% set counter = {'dns': 6,'ip': 1,} %}{% macro increment(dct, key, inc=1)%}{% if dct.update({key: dct[key] + inc}) %} {% endif %}{% endmacro %}[req]
+req_extensions = v3_req
+distinguished_name = req_distinguished_name
+[req_distinguished_name]
+[ v3_req ]
+basicConstraints = CA:FALSE
+keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+subjectAltName = @alt_names
+[alt_names]
+DNS.1 = kubernetes
```

Comment:
I think these dns entries are only used on the masters, as they are for the api server to handle.

I think you can skip DNS 1 to 3, and reset the counter in the top which is 6